index.html

```diff
@@ -26,35 +27,42 @@
       }
     }
 
-    async function init(token) {
-      const sql = new cartoToolkitSql.SQL(
-        'roman-carto',
-        oauth.token
-      );
+    let cs;
+    let sql;
 
-      function getFile(rows = 7) {
-        const contents = ['id,name'];
+    function getFile(rows = 7) {
+      const contents = ['id,name'];
 
-        for (var i = 0; i < rows; i++) {
-          contents.push(`${i},A`);
-        }
-
-        return contents.join('\n');
+      for (var i = 0; i < rows; i++) {
+        contents.push(`${i},A`);
       }
 
-      async function copyFile(rows = 7) {
-        await sql.query('truncate copy_test');
+      return contents.join('\n');
+    }
+
+    async function copyFile(rows = 7) {
+      await sql.create('copy_test', ['id numeric', 'name text']);
 
-        const file = getFile(rows);
+      await sql.query('truncate copy_test');
 
-        const result = await sql.copyFrom(file, 'copy_test', ['id', 'name']);
+      const file = getFile(rows);
 
-        console.log(result);
+      const result = await sql.copyFrom(file, 'copy_test', ['id', 'name']);
 
-        const data = await sql.query('SELECT * FROM copy_test LIMIT 100');
+      console.log(result);
 
-        return data;
-      }
+      const data = await sql.query('SELECT * FROM copy_test LIMIT 100');
+
+      return data;
+    }
+
+    async function init(token) {
+      sql = new cartoToolkitSql.SQL(
```

Comment:
We should probably agree on all these 'rollup' names for umd, as they are going to be the 'public face' of the library outside, and it would be hard to change them in the future

Reply:
https://github.com/CartoDB/toolkit/issues/15